examples/tests/valid/resource.w

```diff
@@ -3,11 +3,16 @@ bring cloud;
 // User defined resource
```

Comment:
Can we add an example where a user defined resource has an inflight init function, and it has a child resource that also has an inflight init function (and validate that it's initialized properly etc.)? It's important to check that these behaviors work with more than one layer of composition

Reply:
I agree, but we currently don't support a user defined resource inheriting from another user defined resource because of no `super()` support: #1123
I'll add a comment about that in the relevant issue.

Reply:
I think inheritance may be a separate use case than having one resource containing another resource

Reply:
> I think inheritance may be a separate use case than having one resource containing another resource

I think that case is covered in the _resource.w_ test.

Reply:
You're right, I missed it